config/routes.rb

```diff
@@ -4,13 +4,17 @@
   root 'static_pages#home'
   get 'myscope' => 'scope#myscope'
   get 'help'    => 'static_pages#help'
+
   resources :languages
 
   resources :countries
 
   resources :percentages
 
-  resources :schools, only: [:index, :edit, :show]
+  resources :schools, only: [:index, :show]
+  get 'offers'   => 'schools#edit'
+  put 'offers'   => 'schools#update'
+  patch 'offers' => 'schools#update'
```

Comment:
This seems a bit weird to me, usually this is done for custom routes you want to map controller actions too. Why not just use the ones that rails will give you with `resources`?